google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/util/GctTracking.java

```diff
@@ -47,4 +47,11 @@
       "cloud.debugger.close.continue.listen";
   public static final String CLOUD_DEBUGGER_NOTIFY_BREAKPOINT_LIST_CHANGE =
       "cloud.debugger.notify.breakpoint.list.change";
+
+  public static final String APP_ENGINE_OLD_PLUGIN_NOTIFICATION =
+      "appengine.oldplugin.notification";
+  public static final String APP_ENGINE_OLD_PLUGIN_NOTIFICATION_CLICK =
```

Comment:
@aslo I think that the name of this label is too broad. It sounds like you're clicking on the notification, but we want to specifically know that the 'disable plugin' link was clicked.